scripts/apps/search/directives/SearchPanel.js

```diff
@@ -246,20 +246,37 @@ export function SearchPanel($location, desks, privileges, tags, asset, metadata,
                 }
             }
 
+            /*
```

Comment:
This is not the correct documentation syntax. You'll have to check with `grunt docs` if this displays. It might be necessary to document the parents too (SearchPanel). See the examples in the contribution guidelines. Or in the `TranslationService` you wrote 👍